packages/calypso-e2e/src/lib/pages/full-site-editor-page.ts

```diff
@@ -34,10 +36,12 @@ const selectors = {
 	templateLoadingSpinner: '[aria-label="Block: Template Part"] .components-spinner',
 	closeStylesWelcomeGuideButton:
 		'[aria-label="Welcome to styles"] button[aria-label="Close dialog"]',
+	limitedGlobalStylesModalTryButton: '.wpcom-global-styles-modal button:has-text("Try it out")',
```

Comment:
**comment:non-blocking** - this works if the UI is English, but if we ever want to expand FSE testing to non-English then we'd have to adjust the selector. Just a comment, no need to take action.

Reply:
Thanks! I adjusted it in https://github.com/Automattic/wp-calypso/pull/71333/commits/7d92a6b24794b42e8bfee463d697c30774561bf4 so it no longer depends on the English language to be set.